docs/reference/api.yml

```
@@ -0,0 +1,3204 @@
+---
+openapi: 3.0.1
+info:
+  title: Open Food Facts Open API
+  description: As a developer, the Open Food Facts API allows you to get information and contribute to the products database. You can create great apps to help people make better food choices and also provide data to enhance the database.
+  termsOfService: https://openweathermap.org/terms
+  contact:
+    name: Open Food Facts
+    url: https://slack-ssl-openfoodfacts.herokuapp.com/
+    email: contact@openfoodfacts.org
+  license:
+    name: License (MIT, Apache 2.0, etc)
+    url: https://opendatacommons.org/licenses/odbl/summary/index.html
+  version: "2"
+servers:
+- url: https://world.openfoodfacts.org
+  description: prod
+paths:
+  /api/v2/product/{barcode}:
+    get:
+      tags:
+      - Read Requests
+      summary: Get for a specific product by barcode
+      parameters:
```

there are a lot more of possible parameters.

It's ok not to document them now, but:

- we will have to use the equivalent of schema to avoid repeting ourselves (see https://davidgarcia.dev/posts/how-to-split-open-api-spec-into-multiple-files/)
- it's better to add a comment 
 ```suggestion
      # TODO: add all optional parameters
      parameters:
```